.github/workflows/deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy master
+
+on: 
+  push:
+    branches: [ master, matt/hello-world ]
```

Comment:
I set this up to work on a dev branch as well as 'master' to validate that the deploy actually works as expected. Will remove 'matt/hello-world' here.